open-company/open-company-web
# REFACTOR: Org refactor

Card: https://trello.com/c/nOF8CduC

Move all the org related actions to the proper namespace.

Actions involved:
```
:org
:org-create
:org-edit
:org-edit-save
:org-redirect
```

### Tests

Test 1:
- with an incognito window go to /sign-up
- signup with a new email
- insert the org name and a logo
- [x] are you redirected to /{org-slug}/all-posts? Good
- [x] can you see the new org logo at the top? Good

Test 2:
- open the user menu
- click on Team Settings
- change the org name
- save
- [x] can you see a PATCH request? Good
- refresh
- [x] can you see the new name still in the team settings panel? Good
- [x] repeat changing name and logo

Test 3:
- open the user menu
- go to Invite People
- invite a user (admin or contributor)
- [x] check that :org-loaded action is logged in the console
